### ISSUE TITLE
Pickle error and never ending code with 1.43.1

### ISSUE THREAD

**Issue by TabeaW** [sertit/sertit-utils#26]
With the new version 1.43.1 we receive a pickle error while opening Sentinel-3 OLCI data. The error isn't really helpful, as everything comes from the distributed/pickle/python side, the error:
```
distributed.protocol.pickle - ERROR - Failed to serialize <pykdtree.dktree.KDTree object 
...
TypeError: no default __reduce__ due to non-trivial __cinit__
```
After the error, the code hangs and does not end.
There was no error with the version 1.43.0.

**Comment by bastiencyr** [comment]
Hello @TabeaW , 
Do you use threading or cython somwhere in your code ? 
Notice that sertit-utils will not use Dask if you dont have it in your environement

**Comment by bastiencyr** [comment]
The issue is linked to https://github.com/storpipfugl/pykdtree/issues/81

**Comment by bastiencyr** [comment]
rasters now creates a local threaded cluster which should solve your issue. You can test your code against the latest version of sertit-utils (on main branch)

**Comment by remi-braun** [comment]
Done, released in 1.43.2
